appveyor.yml

```diff
@@ -3,39 +3,61 @@ skip_commits:
   message: /^\(?doc\)?.*/
 clone_depth: 10
 init:
-- SET
-- 'mkdir C:\ProgramData\PuppetLabs\code && exit 0'
-- 'mkdir C:\ProgramData\PuppetLabs\facter && exit 0'
-- 'mkdir C:\ProgramData\PuppetLabs\hiera && exit 0'
-- 'mkdir C:\ProgramData\PuppetLabs\puppet\var && exit 0'
+  - SET
+  - 'mkdir C:\ProgramData\PuppetLabs\code && exit 0'
+  - 'mkdir C:\ProgramData\PuppetLabs\facter && exit 0'
+  - 'mkdir C:\ProgramData\PuppetLabs\hiera && exit 0'
+  - 'mkdir C:\ProgramData\PuppetLabs\puppet\var && exit 0'
 environment:
+  PUPPET_GEM_VERSION: ~> 4.0
```

Comment:
I'm confused how this got into the environment.  It's not specified in the sync.yml file

Reply:
This comes from the config_defaults specified in the templates: https://github.com/puppetlabs/pdk-templates/blob/master/config_defaults.yml#L31